scripts/orchestrators/tests.runner.sh

```diff
@@ -118,7 +118,7 @@ bicep() {
   }
 
   # cd to the tests directory
-  pushd ../../IAC/Bicep/test
+  pushd ../../IAC/Bicep/tests
```

Comment:
Can we please revert the test folder name back to test (singular) so we have parity with Terraform?

Reply:
good point!